File: Java/projetoFuncionario/src/entities/Funcionario.java
package entities;

public class Funcionario {
	public String nome;
	public double salario;
	public double imposto;

	public double salarioReal() {

		return salario - imposto;

	}

	public void aumentaSalario(double porcentagem) {

		salario += salario * porcentagem / 100;

	}

	public String toString() {

		return nome + ", R$" + String.format("%.2f", salarioReal());
	}

}

File: C#/projetoFuncionarios/projetoFuncionarios/Funcionario.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace projetoFuncionarios
{
    class Funcionario
    {
        public String Nome { get; set; }
        public int Id { get; set; }
        public double Salario { get; set; }

        
        public Funcionario(string nome, int id, double salario)
        {
            Nome = nome;
            Id = id;
            Salario = salario;
        }

        public void Aumentar( double porcentagem)
        {
             Salario += Salario * (porcentagem/100);
        }

        public override string ToString()
        {
            return "Id: " + Id + 
                   "\nNome: " + Nome +
                   "\nSalario: " + Salario.ToString("F2") +
                   "\n";
        }
    }
}
